app/models/garden.rb

```diff
@@ -24,6 +25,13 @@ class Garden
   embeds_many :pictures, cascade_callbacks: true, as: :photographic
   accepts_nested_attributes_for :pictures
 
+  track_history   on: :location,
```

Comment:
Will the location of a garden likely change? Does it make sense to track the history of the garden? Is it needed for this PR?